build/bundle.js

```diff
@@ -20949,6 +20949,8 @@ var FCC_Global =
 
 	var _globalD3Tests = __webpack_require__(54);
 
+	var _alignmentTestsD = __webpack_require__(56);
```

Comment:
@tchaffee @no-stack-dub-sack Can someone tell me why webpack is truncating the name of alignmentTestsD3 to _alignmentTestsD and / or let me know if this is cause for concern?